res/css/views/rooms/_EntityTile.scss

```diff
@@ -69,8 +69,6 @@ limitations under the License.
     padding-right: 12px;
     padding-top: 4px;
     padding-bottom: 4px;
-    width: 36px;
```

Comment:
Is the idea here that the size is being set with JS which ends up using the `toRem` function?

Reply:
Yeah, the BaseAvatar component takes width and height props.